Makefile

```diff
@@ -4,53 +4,83 @@ include $(abspath $(PROJECT_DIR)/build/automation/init.mk)
 # ==============================================================================
 # Development workflow targets
 
-setup: # Set up project
-	make project-config
-	make python-virtualenv
-	pip install -r application/requirements.txt
-
-build-dev: # Build dev requirements
+setup: project-config # Set up project
 	make docker-build NAME=serverless
 	make serverless-requirements
 	make python-requirements
 
 build: # Build lambdas
-	make event-sender-build
+	make event-sender-build AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
 
-stop: project-stop # Stop project
+start: # Stop project
+	make project-start AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
+
+stop: # Stop project
+	make project-stop AWS_ACCOUNT_ID_MGMT=$(AWS_ACCOUNT_ID_NONPROD)
 
 restart: stop start # Restart project
 
 log: project-log # Show project logs
 
 deploy: # Deploys whole project - mandatory: PROFILE
+	eval "$$(make populate-deployment-variables)"
+	make terraform-apply-auto-approve STACKS=lambda-security-group
+	make serverless-deploy
+
+quick-deploy: # Deploys all lambdas - mandatory: PROFILE
```

Comment:
Think we need a better name for this, to describe it only deploys the serverless stuff e.g. `sls-only-deploy`